lib/ui/text.dart

```diff
@@ -2123,7 +2123,20 @@ class ParagraphBuilder extends NativeFieldWrapperClass2 {
 Future<void> loadFontFromList(Uint8List list, {String fontFamily}) {
   return _futurize(
     (_Callback<void> callback) => _loadFontFromList(list, callback, fontFamily)
-  );
+  ).then(_sendFontChangeMessage);
+}
+
+final ByteData _fontChangeMessage = utf8.encoder.convert(
```

Comment:
This could also be static, to prevent creating multiple instances

Reply:
This is not in a class. Do you mean put this field to be a static member for some class? I am not familiar with how dart compile the code, I thought define it in the scope of this file is the same thing as a static member of a class?

Reply:
Oh, reading fail on my part. it is fine as a top level